scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -4,8 +4,8 @@
 #  gatk4_jar: java jar file containing gatk 4 (protected)
 #  intervals: genomic intervals
 #  ref_fasta, ref_fasta_index, ref_dict: reference genome, index, and dictionary
-#  tumor_bam, tumor_bam_index, and tumor_sample_name: self-explanatory
-#  normal_bam, normal_bam_index, and normal_sample_name: self-explanatory
+#  tumor_bam, tumor_bam_index: self-explanatory
```

Comment:
I believe Geraldine has some good documentation for putting at the head of the M2 WDL.  We should probably bring that into the GATK repo.  Will it need edits, @vdauwera ?

Reply:
@davidbenjamin Did you do this?